coffee_github/src/repository.rs

```diff
@@ -116,22 +117,53 @@ impl Github {
-                                "go.mod" => PluginLang::Go,
-                                "cargo.toml" => PluginLang::Rust,
-                                "pubspec.yaml" => PluginLang::Dart,
-                                "package.json" => PluginLang::JavaScript,
-                                "tsconfig.json" => PluginLang::TypeScript,
-                                _ => PluginLang::Unknown,
-                            };
+                            match file_name {
+                                "requirements.txt" => {
+                                    plugin_lang = PluginLang::PyPip;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.py"))
+                                }
+                                "pyproject.toml" => {
+                                    plugin_lang = PluginLang::PyPoetry;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.py"))
+                                }
+                                "go.mod" => {
+                                    plugin_lang = PluginLang::Go;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.go"))
+                                }
+                                "cargo.toml" => {
+                                    plugin_lang = PluginLang::Rust;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.rs"))
+                                }
+                                "pubspec.yaml" => {
+                                    plugin_lang = PluginLang::Dart;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.dart"))
+                                }
+                                "package.json" => {
+                                    plugin_lang = PluginLang::JavaScript;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.js"))
+                                }
+                                "tsconfig.json" => {
+                                    plugin_lang = PluginLang::TypeScript;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.ts"))
+                                }
+                                _ => {
+                                    plugin_lang = PluginLang::Unknown;
+                                    path_to_plugin =
+                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.ts"))
+                                }
```

Comment:
```suggestion
                                    plugin_lang = PluginLang::Unknown;
                                    path_to_plugin =
                                        Some(format!("{tmp_root_path}/{tmp_plugin_name}.ts"))
                                }
```

Monkey copy and paste?